src/front/DataInspector.js

```diff
@@ -124,3 +141,24 @@ L.Kosmtik.DataInspector = L.Class.extend({
     }
 
 });
+
+L.FormBuilder.LabeledCheckBox = L.FormBuilder.CheckBox.extend({
```

Comment:
This would better land as an option of default Checkbox builder, but I can manage this later :)